gulpfile.js

```diff
@@ -10,6 +10,8 @@ const git = require('gulp-git-streamed');
 const gulp = require('gulp');
 const gulpif = require('gulp-if');
 const gutil = require('gulp-util');
+const lockfileVersions = [1];
+const packageLock = require('./package-lock.json');
```

Comment:
I would remove this and instead rely on `gulp.src`

Reply:
@mattolson we will need that outside the scope that use the src, to validate the lockfileVersion

Reply:
Do we need to read it in up here, or can we do that in `bumpTask`?

Reply:
@lord2800 Can do it either way, we use it only in bumpTask anyway. I can change that if @mattolson will accept my previous comment related to not removing that completely.